lib/SILGen/SILGenLValue.cpp

```diff
@@ -2794,6 +2794,10 @@ static ManagedValue visitRecNonInOutBase(SILGenLValue &SGL, Expr *e,
     ctx = SGFContext::AllowGuaranteedPlusZero;
   }
 
+  //if (options.MustBorrowBase) {
+  //  ctx = SGFContext::AllowGuaranteedPlusZero;
```

Comment:
@jckarter this was one attempt to force this... but I found that doing https://github.com/apple/swift/pull/62872/files#diff-a1fd3272e422f41d7025b94476402d6154a4784fe8e78895921eb888e69fae20R1051 works as well and seems cleaner.